app/Http/Frontend/Controllers/HomeController.php

```diff
@@ -25,6 +26,33 @@ public function __construct()
      */
     public function index()
     {
-        return view('frontend.dashboard.index');
+        $products = Product::select('id', 'name', 'price', 'image')->orderBy('id', 'DESC')->skip(0)->take(6)->get();
+        return view('frontend.dashboard.index', compact('products'));
+    }
+
+    /**
+     * Show list products.
+     *
+     * @param int $id id category
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function listProducts($id)
```

Comment:
change HomeController@index-> ProductsController@indexFrontend ?  How to config ? Please!